docs/sources/dashboards/create-reports/_index.md

```diff
@@ -0,0 +1,331 @@
+
+- **Include table data as PDF appendix** - Adds an appendix to the main dashboard PDF.
+- **Attach a separate PDF of table data** - Generates a separate PDF file.
+
+This feature relies on the same plugin that supports the [image rendering](ref:image-rendering) features.
+
+### 3. Schedule
+
+At this step, set scheduling information. Options vary depending on the frequency you select.
+
+<!-- prettier-ignore-start -->
+
+| Option | Description |
+| ------ | ----------- |
+| Frequency | Scheduled reports can be sent once, or repeated on an hourly, daily, weekly, or monthly basis, or sent at custom intervals. You can also disable scheduling by selecting **Never**, for example to send the report using the API. |
+| Time | Choose one of the following:<ul><li>**Send now** sends the report immediately after you save it. To stop sending the report at some point in the future, add an end date.</li><li>**Send later** schedules a report for a later date. When you select this option, the required **Start date**, **Start time**, and **Time zone** options are displayed.</li></ul> |
+| End date | If you leave the end date empty, the report is sent out indefinitely. |
+| Send only from Monday to Friday | For reports that have an hourly or daily frequency, you can choose to send them only from Monday to Friday. |
+| Send on the last day of the month | When you schedule a report with a monthly frequency, and set the start date between the 29th and the 31st of the month, the report is only sent during the months that have those dates. If you want the report to be sent every month, select this option instead. This way, the report is sent on the last day of every month regardless of how many days there are in any given month. |
+
+<!-- prettier-ignore-end -->
+
+### 4. Share
+
+At this step, enter information related to sharing the report:
+
+<!-- prettier-ignore-start -->
+
+| Option | Description |
+| ------ | ----------- |
+| Report name | Name of the report as you want it to appear in the **Reports** list. The report name also populates the email subject line. |
+| Recipients | Enter the email addresses of the people or teams that you want to receive the report, separated by commas or semicolons. |
+| Reply-to email address | The address that appears in the **Reply to** field of the email. |
+| Message | Message body in the email with the report. |
+| Include a dashboard link | Include a link to the dashboard from within the report email. |
+
+<!-- prettier-ignore-end -->
+
+Click **Send test email** in the top-right corner of the screen to verify that the configuration works as expected and to verify that emails are working. You can choose to send this email to the recipients configured for the report, or to a different set of email addresses only used for testing. The last saved version of the report will be sent to selected emails.
```

Comment:
```suggestion
Click **Send test email** in the top-right corner of the screen to verify that the configuration works as expected and to verify that emails are working. You can choose to send this email to the recipients configured for the report, or to a different set of email addresses only used for testing.
```
This is not true anymore. If you make changes to the report and send a test email, it will include those even if you didn't save the report.